beta-src/src/components/WDActionIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { useState } from "react";
+import { SvgIcon, Box } from "@mui/material";
+import { ActionIcon, ActionIconSelected } from "./svgr-components";
+
+const WDActionIcon: React.FC = function () {
+  const [active, setActive] = useState(false);
+
+  const highlightActionHandler = () => {
+    setActive(!active);
```

Comment:
setActive(isActive ? false : true)

is more precise and preferred.

Reply:
For this change, when I use the ternary I get this error from esLint: Unnecessary use of boolean literals in conditional expression.eslint[no-unneeded-ternary](https://eslint.org/docs/rules/no-unneeded-ternary).

When I prompt the "quick fix" action, it changes the code back to (!active). 

If need be, I can disable the error and keep your changes! It just adds a comment above the line allowing it stay.

Reply:
ok let's leave it as is then. FB stack is different than airbnb rules apparently and that is fine. go with !active